frontend/src/routes/email/tester/+page@.svelte

```diff
@@ -1,11 +1,51 @@
 <script lang="ts">
-  import type { PageData } from './$types';
+  import { onMount } from 'svelte';
+  import { EmailTemplate } from '../emails';
 
-  export let data: PageData;
+  export const templates = [
+    {
+      type: EmailTemplate.ForgotPassword,
+      resetUrl: 'https://garfield.com',
+    },
+    {
+      type: EmailTemplate.VerifyEmailAddress,
+      verifyUrl: 'https://google.com',
+    },
+    {
+      label: 'Verify New Email Address',
+      type: EmailTemplate.VerifyEmailAddress,
+      verifyUrl: 'https://google.com',
```

Comment:
maybe make this go to `/user?changedEmail` or whatever it ends up being so you can actually test out that page without having to do the full workflow. Same with the one above but for the verify case.